docs/ref/unicode.txt

```diff
@@ -1,18 +1,19 @@
 ============
-Unicode data
+Encoded data
 ============
 
-Django natively supports Unicode data everywhere. Providing your database can
-somehow store the data, you can safely pass around strings to templates,
+Django uses the concepts of text and bytes when dealing with data.
```

Comment:
I'm not sure if "text" is the best term. For example, Python's documentation [seems to use "string"](https://docs.python.org/3/howto/unicode.html#the-string-type).

Instead of "Django uses the concepts" would it be better to explain that these are Python concepts?